docs/contributing/translator_onboarding.md

```diff
@@ -0,0 +1,35 @@
+# ONBOADING GUIDE FOR TRANSLATORS
+
+   Wagtail uses transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.

```

Comment:
Just remove any added whitespace on these and other lines.

```suggestion
Wagtail uses Transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
```
`Transifex` is a proper noun so should always have a capital T